pkg/controller/util_test.go

```diff
@@ -285,7 +285,7 @@ var _ = Describe("ValidateClone", func() {
 
 		err := ValidateClone(sourcePvc, dvSpec)
 		Expect(err).To(HaveOccurred())
-		Expect(err.Error()).To(ContainSubstring("target resources requests storage size is smaller than the source"))
+		Expect(err.Error()).To(ContainSubstring("is smaller than the source"))
```

Comment:
if possible maybe you should check the full err including the the values that you added to the log? it will give a fuller cover to the tests